Package/gulpfile.js

```diff
@@ -210,6 +213,27 @@ const copyVCXProjUWPFiles = () => {
     .pipe(gulp.dest('Assembled-Windows/windows/BabylonReactNative'));
 }
 
+const copyOpenXRInfoFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Dependencies/xr/Dependencies/OpenXR-MixedReality/LICENSE')
+  .pipe(gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Dependencies/xr/Dependencies/OpenXR-MixedReality/README.md'))
+  .pipe(gulp.dest('Assembled-Windows/windows/OpenXR-MixedReality'));
```

Comment:
@bghgary @ryantrem @tymlipari I am not sure if there are concerns around distributing headers from submodule dependencies. If this needs a separate npm package let me know :)

Reply:
Chatted with folks offline, headers are coming from another MIT license repo and should be fine.